### PR TITLE
[UWP] Implement bleed

### LINKED ISSUE #2357: [UWP] bleed property on elements
**Automatically generated task. Any updates to the title and body will be overwritten. Do NOT modify the labels.**

UWP implementation of #2109

### PR DESCRIPTION
**Pull request by RebeccaAnne**
Fixes #2357

How verified: Updated the UWP test files and confirmed correct bleeding behavior.

### CODE REVIEW

**Comment by shalinijoshi19** [comment]
Nice! Quick ask: Could you start including (and enforcing in code reviews) a quick Description section that includes a "How Verified" ? Thanks

**Comment by shalinijoshi19** [comment]
Yikes -- indeed that's a *lot* of files!!! I'd be curious to know of the run times today and the potential gains we are looking at by trimmimg and pruning the data set

**Comment by RebeccaAnne** [comment]
By input data sets you mean the test cards? What did you have in mind? 

---
In reply to: [481902610](https://github.com/Microsoft/AdaptiveCards/pull/2662#issuecomment-481902610) [](ancestors = 481902610)

**Comment by RebeccaAnne** [comment]
Anyone know if there's a way to get CodeFlow to cooperate when there are so many files?

**Comment by shalinijoshi19** [comment]
Btw this isnt the UWP implementation for Bleed is it? It's the test fixes? Accurate? If so could you update the description real quick?

**Comment by RebeccaAnne** [comment]
> Btw this isnt the UWP implementation for Bleed is it? It's the test fixes? Accurate? If so could you update the description real quick?

This is the UWP implementation. Unfortunately it looks like CodeFlow has a limit on how many files it can handle. The code is part of this PR, but it isn't showing up on the CodeFlow.

File: source/uwp/Renderer/lib/AdaptiveColumn.cpp [hunk 1]
```diff
@@ -124,6 +133,26 @@ namespace AdaptiveNamespace
...
+
+    HRESULT AdaptiveColumn::put_Bleed(boolean isBleed)
+    {
+        m_bleed = isBleed;
+        return S_OK;
+    }
+
+    HRESULT AdaptiveColumn::get_BleedDirection(ABI::AdaptiveNamespace::BleedDirection* bleedDirection)
+    {
+        // TODO: Current behavior is broken because it doesn't update when bleed updates. Unfortunately, neither does
```
Comment (almedina-ms) [inline source/uwp/Renderer/lib/AdaptiveColumn.cpp]: do we have an issue to track this?

File: source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -29,14 +29,14 @@
 		6B096D5122551F35006CC034 /* RichTextBlock.TextRun.SelectActions.json in Resources */ = {isa = PBXBuildFile; fileRef = 6B096D5022551F35006CC034 /* RichTextBlock.TextRun.SelectActions.json */; };
 		6B096D53225585DD006CC034 /* RichTextBlock.Highlights.json in Resources */ = {isa = PBXBuildFile; fileRef = 6B096D52225585DD006CC034 /* RichTextBlock.Highlights.json */; };
 		6B096D582257DA0C006CC034 /* Action.IconUrl.TwoActions.json in Resources */ = {isa = PBXBuildFile; fileRef = 6B096D572257DA0B006CC034 /* Action.IconUrl.TwoActions.json */; };
+		6B096D6C225EB726006CC034 /* Bleed.Comprehensive.json in Resources */ = {isa = PBXBuildFile; fileRef = 6B096D6B225EB726006CC034 /* Bleed.Comprehensive.json */; };
```
[the remark this answers is not in the document]
Comment (RebeccaAnne) [reply source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer.xcodeproj/project.pbxproj]: Removing the test files causes a break in iOS. I asked joseph to update iOS for me to unbreak it and reflect the changes I made to the test files.

File: source/uwp/Renderer/lib/AdaptiveColumn.cpp (hunk 1 above)
Comment (RebeccaAnne) [reply source/uwp/Renderer/lib/AdaptiveColumn.cpp]: Good point. Logged #2678

**Pull request merged**